apis/v1alpha1/generator.yaml

```diff
@@ -31,7 +31,6 @@ ignore:
     # always calculate it by doing a len(items)...
     - CreateDistributionInput.DistributionConfig.Origins.Quantity
     - CreateDistributionInput.DistributionConfig.Origins.Items.Origin.CustomHeaders.Quantity
-    - CreateDistributionInput.DistributionConfig.Origins.Items.Origin.VpcOriginConfig
```

Comment:
small small nit, i always prefer commenting instead of deleting items in the ignore list, can be very handy when it comes to debugging things...